server.go

```diff
@@ -140,7 +140,7 @@ type ProvisionReq struct {
 	Nameservers []string `json:"nameservers"`
 }
 
-var nameserverPattern = regexp.MustCompile(`\Ans-\d+\.awsdns-\d+(\.[a-z]{1,3}){1,2}\z`)
+var nameserverPattern = regexp.MustCompile(`\Ans-\d+(\.[a-z]{1,3}){1,3}\z`)
```

Comment:
What is the format of the DO nameservers? can we just add a separate pattern that's OR'ed?